src/DropdownItem.js

```diff
@@ -42,7 +37,7 @@ class DropdownItem extends React.Component {
       this.props.onClick(e);
     }
 
-    if (this.props.toggle) {
+    if (this.props.toggle || true) {
```

Comment:
Here too, I think we can use [??](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)